dist/platforms/ubuntu/entrypoint.sh

```diff
@@ -3,12 +3,14 @@
 # Ensure machine ID is randomized for personal license activation
 if [[ "$UNITY_SERIAL" = F* ]]; then
   echo "Randomizing machine ID for personal license activation"
-  dbus-uuidgen > /etc/machine-id && mkdir -p /var/lib/dbus/ && ln -sf /etc/machine-id /var/lib/dbus/machine-id
```

Comment:
This space was likely removed by mistake? It's now formatted differently from builder. Since we're using that as a reference let's not change this line here.

Reply:
Oh, my bad, this was indeed a mistake. I'll change it back.

Reply:
Should be fixed now, thanks for the quick review!